### ISSUE TITLE
Unable to install latest version of radiant

### ISSUE THREAD

**Issue by techno-ballu** [vnijs/radiant#29]
I am facing issue in installing dependency - data.tree. Here is the complete log:
> also installing the dependency ‘data.tree’
trying URL 'http://vnijs.github.io/radiant_miniCRAN/src/contrib/data.tree_0.1.9.tar.gz'
Content type 'application/octet-stream' length 898890 bytes (877 KB)
==================================================
downloaded 877 KB
* installing *source* package ‘data.tree’ ...
** R
** data
** demo
** inst
** preparing package for lazy loading
Error in R6Class("Node", lock_objects = FALSE, public = list(parent = NULL,  : 
  unused argument (lock_objects = FALSE)
Error : unable to load R code in package ‘data.tree’
ERROR: lazy loading failed for package ‘data.tree’
* removing ‘/home/bolaka/R/x86_64-pc-linux-gnu-library/3.2/data.tree’
Warning in install.packages :
  installation of package ‘data.tree’ had non-zero exit status
ERROR: dependency ‘data.tree’ is not available for package ‘radiant’
* removing ‘/home/bolaka/R/x86_64-pc-linux-gnu-library/3.2/radiant’
Warning in install.packages :
  installation of package ‘radiant’ had non-zero exit status

**Comment by vnijs** [comment]
Strange. I just updated a server and had no issues. Could you try installing data.tree from github?

```r
devtools::install_github("gluc/data.tree")
```

**Comment by techno-ballu** [comment]
Hey, thanks for the prompt reply!
Yes installing data.tree from github worked like a charm :)
I am able to install the latest radiant now :+1:

**Comment by vnijs** [comment]
great! I'll update the package in the radiant install repo. Thanks for reporting the issue

**Comment by bjornerstedt** [comment]
When I try to install Radiant on Rstudio Version 1.0.143 (R version 3.4.0) it does not seem to find radiant.data:

```
install.packages("radiant", repos = "https://radiant-rstats.github.io/minicran/", type = "binary")
Warning in install.packages :
  dependency ‘radiant.data’ is not available
trying URL 'https://radiant-rstats.github.io/minicran/bin/macosx/el-capitan/contrib/3.4/radiant_0.8.0.tgz'
Content type 'application/octet-stream' length 466577 bytes (455 KB)
==================================================
downloaded 455 KB

```

**Comment by vnijs** [comment]
Hi Jonas,

That is odd. Could you try?

install.packages("radiant.data", repos = "
https://radiant-rstats.github.io/minicran/", type = "binary")

Vincent

**Comment by bjornerstedt** [comment]
Hello,

Having failed with R 3.4.0 several times, I installed R 3.3.3 and then the
Radiant install worked. When I now tried reinstalling R 3.4.0 and removing
the previous installed packages with remove.package, it still works.

/Jonas

PS. I really like the modularity of Radiant. I noticed that I can get a
pruned down version with radiant.model::radiant.model(), although I could
not find any documentation of this feature. This configuration fits my
econometrics teaching better.

2017-05-05 1:14 GMT+02:00 Vincent Nijs <notifications@github.com>:
